assets/themes/dog/dog.json

```diff
@@ -0,0 +1,427 @@
+                "then": "<div style='background: white; padding: 0.25em; border-radius:0.5em'>{name}</div>"
+              }
+            ]
+          }
+        },
+        {
+          "#": "Rendering of a line",
+          "color": "#ff0",
+          "width": 5
+        }
+      ],
+      "tagRenderings": [
+        {
+          "id": "dogpark-fenced",
+          "mappings": [
+            {
+              "if": "barrier=fence",
+              "then": {
+                "en": "fenced: Yes",
+                "da": "indhegnet: Ja"
+              }
+            },
+            {
+              "if": "barrier=no",
+              "then": {
+                "en": "fenced: No",
+                "da": "indhegnet: Nej"
+              }
+            }
+          ],
+          "question": {
+            "en": "It this dog park fenced in?",
+            "da": "Er denne hundeskov indhegnet?"
+          }
+        },
+        {
+          "id": "bin",
+          "mappings": [
+            {
+              "if": "bin=yes",
+              "then": {
+                "en": "waste bin: Yes",
+                "da": "skraldespand: Ja"
+              }
+            },
+            {
+              "if": "bin=no",
+              "then": {
+                "en": "waste bin: No",
+                "da": "skraldespand: Nej"
+              }
+            }
+          ],
+          "question": {
+            "en": "Does this dog park have a waste bin?",
+            "da": "Har denne hundeskov en skraldespand?"
+          }
+        },
+        {
+          "id": "smalldogs",
+          "mappings": [
+            {
+              "if": "small_dog=separate",
+              "then": {
+                "en": "Have separate area for puppies and small dogs",
+                "da": "Har en indhegning til hvalpe og sm&aring; hunde"
+              }
+            },
+            {
+              "if": "small_dog=shared",
+              "then": {
+                "en": "Does <strong>not</strong> have a separate area for puppies and small dogs",
+                "da": "Har <strong>ikke</strong> en indhegning til hvalpe og sm&aring; hunde"
+              }
+            }
+          ],
+          "question": {
+            "en": "Does this dog park have a separate fenced in area for small dogs and puppies?",
+            "da": "Har denne hundeskov et separat indhegnet omr&aring;de for sm&aring; hunde og hvalpe (hvalpeg&aring;rd)?"
+          }
+        },
+        {
+          "question": {
+            "en": "What is the name of this dog park?",
+            "da": "Hvad hedder denne hundeskov?"
+          },
+          "render": {
+            "en": "The name of this dog park is {name}",
+            "de": "Denne hundeskov hedder {name}"
+          },
+          "freeform": {
+            "key": "name"
+          },
+          "id": "Name"
+        },
+        {
+          "id": "dogarea",
+          "render": {
+            "en": "areal: {_surface:ha} ha",
```

Comment:
Another small nitpick: this would better be prhased as `This dogpark is {_surface:ha} ha big` as it is friendlier for humans.